modules/031-ceph-csi/hooks/get_crds.go

```diff
@@ -174,8 +174,8 @@ func isStorageClassChanged(input *go_hook.HookInput, scName, scReclaimPolicy str
 	for _, storageClass := range storageClasses {
 		sc := storageClass.(StorageClassFilter)
 		if sc.Name == scName {
-			// Annotation check and annotation in templates can be safely removed after release 1.43
-			_, migrationCompleted := sc.Annotations["migration-secret-name-changed"]
```

Comment:
Can we keep previous migration?

Reply:
In this case, it doesn't make sense. This and the previous migration removes the storageclass.

Reply:
There is no check here which parameter has changed.